pkg/backup/s3/s3.go

```diff
@@ -148,3 +146,49 @@ func (s *S3) CopyPrefix(from string) error {
 	}
 	return nil
 }
+
+// LoadCredentials reads the default profile section from a aws credentials file
+func LoadCredentials(data []byte) (creds *credentials.Value, err error) {
+	config, err := ini.Load(data)
+	if err != nil {
+		return
+	}
+	iniProfile, err := config.GetSection("default")
```

Comment:
This looks pretty fragile. We are trying to understand AWS config internal thing here.

Is this the AWS SDK suggested way? Can you point me to any reference?